drivers/crownstone/driver.js

```diff
@@ -121,37 +112,36 @@ class CrownstoneDriver extends Homey.Driver {
 		session.setHandler('list_devices', async (data) => {
 			let deviceList = [];
```

Comment:
Why does this not use the cache?

Reply:
> Why does this not use the cache?

This needs some more love, but there are basically two routes to fill in the data structures in the app:

1. via app boot and 
2. via first time configuration after install (that is in `onPair`). 

In the `onPair()` route we should be able to list devices as fast as possible. This process takes actually quite long and keeps the user waiting. 

Other stuff w.r.t. users, presence etc. is fine to be loaded later. 

Of course, the Homey persists devices etc. over reboots. It's just trying to err on the side of safety to obtain stuff from our cloud on that moment.